backend/api/v1/schema_design_service.go

```diff
@@ -673,27 +673,47 @@ func (s *SchemaDesignService) convertSheetToSchemaDesign(ctx context.Context, sh
 
 	engine := v1pb.Engine(sheet.Payload.SchemaDesign.Engine)
 	schemaDesignType := v1pb.SchemaDesign_Type(sheet.Payload.SchemaDesign.Type)
+	// For backward compatibility, we default to MAIN_BRANCH if the type is not specified.
+	if schemaDesignType == v1pb.SchemaDesign_TYPE_UNSPECIFIED {
+		schemaDesignType = v1pb.SchemaDesign_MAIN_BRANCH
+	}
 	name := fmt.Sprintf("%s%s/%s%v", common.ProjectNamePrefix, project.ResourceID, common.SchemaDesignPrefix, sheet.UID)
 
+	var baselineSheetName string
+	if schemaDesignType == v1pb.SchemaDesign_MAIN_BRANCH {
```

Comment:
if a && b {
} else if {
}

Reply:
Moving code from https://github.com/bytebase/bytebase/pull/9017/files#diff-b83cfbdb2860fcfb4d75f7a8aa9256c56a9e3a5afe63adb83f78bde119b58234L728. Trying to be consistent.